src/packages/client/src/utils/generateInFolder.ts

```diff
@@ -70,13 +72,21 @@ export async function generateInFolder({
 
   const platform = await getPlatform()
 
+  let runtimePath
+  if (useLocalRuntime) {
+    runtimePath = path.relative(outputDir, path.join(__dirname, '../runtime'))
+  }
+  if (useBuiltRuntime) {
```

Comment:
because it's 2 if and not else if, this makes me think that both could be true, is it possible?

Reply:
Exactly, both can be true.
So possible cases:
`--skip-transpile` - just true for first `if`
`--skip-transpile --built-runtime` - goes into second.

But you know what, I want the second to only run when the first one is provided. So I'll add a check there.